src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -86,7 +93,7 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
               coveringIndexProperties)),
           Content.fromDirectory(absolutePath, fileIdTracker),
           Source(SparkPlan(sourcePlanProperties)),
-          Map())
+          projectVersion.toMap)
 
       case None => throw HyperspaceException("Invalid plan for creating an index.")
```

Comment:
Can you also update `IndexLogEntryTest`?

Reply:
Done, added some tests